### ISSUE TITLE
About segment fault under debian10 gnome(wayland) with gtk3+

### ISSUE THREAD

**Issue by fengjl026** [wxMEdit/wxMEdit#196]
Recently I installed debian10.3 with virtualbox. I compiled wxmedit with the source code gtk3+. The reason for choosing gtk3 is that I used the wayland gnome environment, and found a segment fault, and solved it with the following code modification. This modification replaced the original portability Bad code, replaced with gdk-related functions, the test can run under gnome(wayland), weston.

---

最近我用virtualbox 安装了debian10.3 . 用源码gtk3+编译了wxmedit. 选择gtk3的原因是我使用了wayland gnome的环境, 发现段错误, 用如下代码修改解决, 这次修改替换掉了原先的移植性不好的代码, 换成了gdk相关的函数, 测试可以在 gnome(wayland), weston下运行
[wxmedit_app.cpp.txt](https://github.com/wxMEdit/wxMEdit/files/4619498/wxmedit_app.cpp.txt)

**Comment by nerijus** [comment]
Please provide diff -u wxmedit_app.cpp.orig wxmedit_app.cpp, i.e. only the differencies, not the whole file.

**Comment by hltj** [comment]
The Translation

Recently I installed debian10.3 with virtualbox. I compiled wxmedit with the source code gtk3+. The reason for choosing gtk3 is that I used the wayland gnome environment, and found a segment error, and solved it with the following code modification. This modification replaced the original portability Bad code, replaced with gdk-related functions, the test can run under gnome(wayland), weston.
